simbad/command_line/__init__.py

```diff
@@ -321,16 +322,35 @@ def _simbad_contaminant_search(args):
         ed.process_miller_arrays()
         ed.output_mtz(temp_mtz)
 
-    rotation_search = AmoreRotationSearch(args.amore_exe, temp_mtz, args.mr_program, args.tmp_dir,
-                                          stem, args.max_contaminant_results)
+    if args.rot_program.lower() == "amore":
+        from simbad.rotsearch import AmoreRotationSearch
 
-    rotation_search.run(os.path.abspath(args.cont_db), nproc=args.nproc, shres=args.shres,
-                        pklim=args.pklim, npic=args.npic,
-                        rotastep=args.rotastep,
-                        min_solvent_content=args.min_solvent_content,
-                        submit_qtype=args.submit_qtype,
-                        submit_queue=args.submit_queue,
-                        chunk_size=args.chunk_size)
+        rotation_search = AmoreRotationSearch(args.amore_exe, temp_mtz, args.mr_program, args.tmp_dir,
```

Comment:
`PhaserRotationSearch` and `AmoreRotationSearch` do not have the same order of input arguments ... this should be respected or made optional keyword arguments.